samples/wifi/raw_tx_packet/README.rst

```diff
@@ -0,0 +1,177 @@
+.. _wifi_raw_tx_packet_sample:
+
+Wi-Fi: Raw TX packet
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Raw TX packet sample demonstrates how to transmit raw IEEE 802.11 packets.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+The sample generates and broadcasts 802.11 beacon frames as raw TX packets.
+As a consequence, the nRF70 Series device can be identified as a Wi-Fi beaconing device.
+
+The sample demonstrates how to transmit raw TX packets in both connected Station and non-connected Station modes of operation.
+The sample provides the option to select the traffic pattern between the following modes:
+
+* Continuous packet transmission.
+* Fixed number of transmitted packets.
+
+The configurations for connected Station or non-connected Station modes, and for continuous or fixed packet transmission, are set at build time.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/raw_tx_packet/Kconfig`):
+
+.. options-from-kconfig::
+
```

Comment:
We can add a line here like, "Additionally, the following configuration options are available for the sample:"
Then we can move the content under these subheadings to avoid redundancy.